### PR TITLE
interfaces/system-packages-doc: allow read-only access to /usr/share/cups/doc-root/ and /usr/share/gimp/2.0/help/

### PR DESCRIPTION
**Pull request by lissyx**
This fixes  https://bugzilla.mozilla.org/show_bug.cgi?id=1770462)

### CODE REVIEW

**Comment by lissyx** [comment]
This is similar to https://github.com/snapcore/snapd/pull/11588, original report was about GIMP's doc, and I came accross cups as a coincidence but why not fix it at the same time ...

**Review by mardy — APPROVED** [review]
Thanks, LGTM!

**Review by alexmurray — CHANGES_REQUESTED** [review]
I think this also needs associated entries in `systemPackagesDocConnectedPlugAppArmor` unless I am missing something?

**Comment by lissyx** [comment]
> I think this also needs associated entries in `systemPackagesDocConnectedPlugAppArmor` unless I am missing something?

Thanks, I've added them

File: tests/main/interfaces-system-packages-doc/task.yaml [hunk 1]
```diff
@@ -13,6 +13,10 @@ prepare: |
     echo text >/usr/share/xubuntu-docs/content
     mkdir -p /usr/share/gtk-doc
     echo text >/usr/share/gtk-doc/content
+    mkdir -p /usr/share/cups/doc-root
+    echo text >/usr/share/cups/doc-root/content
+    mkdir -p /usr/share/gimp/2.0/help
+    echo text >/usr/share/gimp/2.0/help/content
```
Comment (bboozzoo) [inline tests/main/interfaces-system-packages-doc/task.yaml:19]: Thanks for updating the test

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by alexmurray — APPROVED** [review]
LGTM - whilst not relevant for this PR, but it would be good to reduce all the copy-pasta in this interface and instead perhaps have a list of all the various paths that we want to allow access to and then just iterate over this to generate the various AppArmor rules and etc

**Comment by lissyx** [comment]
@mardy @alexmurray So there are a few tests that are failing but I'm unsure why. Also, even though I did sign the CLA, the tooling seems unaware of that. Let me know if I need to do anything more.

**Comment by bboozzoo** [comment]
@lissyx that's ok, the failures are unrelated.

Hm the CLA check keeps failing though. I wonder what's going on. DId you use the same email address as in the commit? Did you receive a confirmation from launchpad the CLA was signed?

**Pull request merged**